local.env.dist

```diff
@@ -2,17 +2,6 @@
 MAILER_USERNAME=
 MAILER_PASSWORD=
 ADMIN_EMAIL=
-SAML_IDP=
-SAML_ENTITY_ID=
-SAML_ADMIN_PASSWORD=
-SAML_SECRET_SALT=
-SAML_CONTACT_NAME=
-SAML_CONTACT_EMAIL=
-SAML_SSO_URL=
-SAML_SLO_URL=
-SAML_CERT_FINGERPRINT=
-SAML_ORG_NAME=
-SAML_ORG_URL=
 FRONT_COOKIE_KEY=akljwelkjfwlkajwfkljeafewf5rgtw2
```

Comment:
Would it be worth changing this dummy value to something more likely to get replaced (such as 'aaaaaa' or 'replace-me-with-a-secure-random-string')? Just a thought.